ansible/roles/primary_master/files/admin.conf

```diff
@@ -1,19 +0,0 @@
-    client-key-data: LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFcEFJQkFBS0NBUUVBeU0yYkpGR1BxMUVUNVRCTVB4RUVnbWk1Sk9leVZLTVpDbmYrd1pVblNjUnpldEJxClkrcXpHK29sMnJZZ3hIaDU2L21PME9KTUJrUmp6Y2tiOUNFRDB5UzhNS1hQTlJDZDRjUlBtN0p0Z3AyZUcza2EKR1Zhcy9RVGNrYzhZWjNkOVFDb2hHNHNabndPQm1lRWpoVVZxUklGZHVvZDlFWVlCWjFYRmlFby9YTHhCdW1ZegpGbGZuUkplb1FWVVpNYjQzbUtORFRpSjZxUElvVlZGRkRjbGdwZlllTks1Ym5jaVE0U2E4azhiMUVWR3NPWEcxCkRRL3FrbHNyMmxjYmJDbFMweU5FMUZRdjB1bXBZaHU1RklCa245WHRoYjdPVVYramwrZ01xbStjeUJnN0s1QTQKUzI4QnNVVHVaVW52TDAvU0JGbFZpNjFZRFUrcTN0YjYrdmtGdHdJREFRQUJBb0lCQURpSHVZOWw3cjVhR3ZIQQp3UlpZSWg3Y2dKTGxMNDBvUlZwdklhYzU2RSthSUxDSWtBc0UvZHR6OWFnMjhjNmQ1Q1VNb3FvbnF5TndpWEJiCmw2VUl6bHpZMEtvWlk5WWFrWGdIbmluRTk1TjM4QlRuNlBNTk5iSWRka01PUEY2VWd3eFhUKzdESURsbDhVeHkKMTE4amhhRTNrajE2K3ppOXN4TjJVQi9vVnlJeDFUdmVkVk9XdU95NmZqQTBhbUhPUmkxRWN1bmg5bk1SY2ZGSQpjRGg0UWEzY1h0ZUtCYU54ZzdpbmVFUkdIRi9GKzVkQm5yZmg2cDgxMm4vSTFFdi9SK0tZRkpBRU93NXJHOUZLClUxcG8yQmcrMHFTRkxSTWQyZ0ZqVW43c3ltSjZxbldVNjQxZjZxK2xXOXFFUm00L3AxeDVrUEZmTDFVV0UzcFQKTGhTejJLRUNnWUVBelRXb254dUo5UUdlR3VTbHlYc3Y0MS9SUU1uelVXalFMY3BTd2ZpQ2RSSFhnUmFZYVdwTApxMnB2WTNtOFhab0sxZ2tpTzBncUpOQXB2elNVcWgwcG9kd1R4ZkJUKzFhSzBVT2RkcFB0MHRsNzBhWWFXNUprClZMTE5MTDRkckxjd2Jia2d2UUpiREU5Mkw4cjZiUlZOTER4L0JYeEdoKzYzb2hiODROVTl2OWtDZ1lFQStvQy8KcTZaL1N6WE5JMjJUbTZua2ZyNEs3SWlBVEt5QUpvU0Y0d0V0emNYU0F4bytnRkhiM3VtSUw5Y0x4cVRJYWRVMwpSd1VocmVDWjZPUnVEVHBwNVE0SjM3MDQyMldyQzFkaFRwQkcxTEcvalpQR1NDd2oxaHRXMFZuQU9vRGtnOUl2CkZaTThNamUxamNxREFCVVk2TGFwaVAzVmF0cUFmbkFmZVNPOENBOENnWUVBcUFZQWpFNkROaDVkODQwMkJmUWMKc09MNnRqN2xRTW8rY3MvUDFVMENuSkxBVHMxT2ZEbnh2V2hIemo2V01sbjM5MnJCSWFNa0JzSisvRFUwYU5tNgpCZkxxS2M5dkpFT25KRk5sc2ZyTDV6SDJSU0hwUVFOcnJVSTk1eDIvdEhUR2JZRUF3bUlVWk04SGFYaWNkZ2JVCkNwMTZMK2FpNVpoYnVaZHk5VEVFLzNrQ2dZRUFxYzVtbnl2QTlER25PaEhJVW1mSzdTTlQ4bHpoWTVnM1VtVFoKQVRWR1lFdHRmZzduT1VMRUlZbjZlRHQwenhYYWlPc3dpbUx2SHBlMm5RbGpsSEg1WUh5d3Nzcm81TWc1TnZubgppMEwvMEtRZzdnNnpCUzl1cDI5bTNZalNacFlDMFZ6RDhBTWJkZ3NQZGV0RisrTlJQRWErQzQvTmdKNVhKZDVKCloydVN4WDhDZ1lBRklVRDBUV2RQUitHdnJUdFc4RTNiUzk2K3RGaGRYYWs1NnJxZnhueVY4bjd0RVp2blk0K20KcFQ2Sll6RUJhdW4weGdQNk5zN3lrd3lhMnBqKzJiZDRWQVV4SGpxRmNKMTBsL0RqZDVUVlN0RUlyNkNzMHNBLwpDQ2dQbkVXM0JJNm5LbllQcTQyQlZFN3JvbnFuME9raHU4QnVyRFdWRWUwbTRSWDZoZzFvd2c9PQotLS0tLUVORCBSU0EgUFJJVkFURSBLRVktLS0tLQo=
```

Comment:
Looks like you deleted the file but still has the key in the history. Hopefully not an issue, as looks like a generated k8's config.

Reply:
Yes I added it to .gitignore and this cluster has already been destroyed so all good :)